### PR TITLE
[16.0][3891][ADD] stock_move_line_auto_fill

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
[3891](https://www.quartile.co/web#id=3891&menu_id=517&cids=3&model=project.task&view_type=form)

### CODE REVIEW

**Review by rinaldifirdaus — APPROVED** [review]

File: stock_move_line_auto_fill/models/auto_fill_operation_mixin.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Copyright 2023 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class AutoFillOperationMixin(models.AbstractModel):
```
Comment (yostashiro) [inline stock_move_line_auto_fill/models/auto_fill_operation_mixin.py]: @AungKoKoLin1997 Is there actually a good point to have this mixin model?  These method are only used in stock.move model IIRC.

Comment (AungKoKoLin1997) [reply stock_move_line_auto_fill/models/auto_fill_operation_mixin.py]: Yes. We don't need to use it. I will remove this model and just add functions in `stock.move`

**Comment by rinaldifirdaus** [comment]
@yostashiro san, kindly help to merge this PR. i have updated the PR with latest oca repo. thanks

**Comment by yostashiro** [comment]
@rinaldifirdaus @AungKoKoLin1997 We should have had just two commits in this PR:

- One for adding the confirmed part of the OCA module
- Another for the changes from it

I have authored to squash and merge the PR this time as the commits were cluttered. Please note the above point next time (this is not critically important but in the context of improving the operation quality).

**Pull request merged**